modules/tester.js

```diff
@@ -1063,7 +1063,7 @@ Tester.prototype.begin = function begin() {
         config = getConfig([].slice.call(arguments)),
         next = function() {
             config.test(this, this.casper);
-            if (this.options.concise)
+            if (!this.options.concise)
```

Comment:
It is about the given file having too many statements.

I am currently working on #1133 which touches this, so you can leave it as it is.